README.md

```diff
@@ -9,6 +9,6 @@ Calico networking for Kubernetes can now be found in the [Calico CNI repository]
 
 We recommend migrating to use the Calico CNI plugin for Kubernetes deployments.  Documentation and guides for using the
 CNI plugin with Kubernetes can be found in the [calico-containers
-repository](https://github.com/projectcalico/calico-containers/blob/master/docs/cni/kubernetes/README.md).
+repository](https://github.com/projectcalico/calico-containers/).
```

Comment:
The Calico documentation has actually moved to http://docs.projectcalico.org, so it's a super dead link :)